CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Main (unreleased)
 
+## v229 (?/??/2021)
```

Comment:
For the changelog version and date I usually just have the entries under "unreleased" and then when I'm ready to release I'll add the version and put the date. It lets people know at a glance if what's on main is unreleased or not.